hello/bin/hello.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "../src/hello.h"
+
+int
+main(int argc, char **argv) {
```

Comment:
I think "bin" means executable binary.
So this directory should have only compiled executable binary files.

(on gulp, dist is bin, src is src)